sos/plugins/python.py

```diff
@@ -21,25 +24,70 @@ class Python(Plugin, DebianPlugin, UbuntuPlugin):
 
     packages = ('python',)
 
+    python_version = "python -V"
+
     def setup(self):
-        self.add_cmd_output("python -V", suggest_filename="python-version")
+        self.add_cmd_output(
+            self.python_version, suggest_filename="python-version"
+        )
         self.add_cmd_output("pip list")
 
 
 class RedHatPython(Python, RedHatPlugin):
 
     packages = ('python', 'python36', 'python2', 'platform-python')
+    option_list = [
+        ('hashes', "gather hashes for all python files", 'slow',
+         False)]
 
     def setup(self):
         self.add_cmd_output(['python2 -V', 'python3 -V'])
         self.add_cmd_output("pip list")
         if isinstance(self.policy, sos.policies.redhat.RHELPolicy) and \
                 self.policy.dist_version() > 7:
-            self.add_cmd_output(
-                '/usr/libexec/platform-python -V',
-                suggest_filename='python-version'
-            )
-        else:
-            self.add_cmd_output('python -V', suggest_filename='python-version')
+            self.python_version = "/usr/libexec/platform-python -V"
+        super(RedHatPython, self).setup()
```

Comment:
This is the correct approach, but now there's a need to remove the separate call to `pip list` (line 45) in `RedHatPython` since it will be collected in the base `Python.setup()` now. Otherwise we'll get duplicate files.

Reply:
Done!